core/frontend/src/tools/Tool.ts

```diff
@@ -696,6 +696,11 @@ export abstract class InteractiveTool extends Tool {
   public reloadToolSettingsProperties() {
     IModelApp.toolAdmin.reloadToolSettingsProperties();
   }
+
+  /** Used to bump the value of a tool setting. If no `settingIndex` param is specified, the first setting is bumped.

```

Comment:
What does it mean to "bump" a tool setting value?

Reply:
Toggle a Boolean or cycle through an Enum’s values. The functionality is usually associated with a keyboard shortcut and is used in conjunction with AccuDraw.

Reply:
Thanks. Can that be expressed in the documentation?

Reply:
Fixed